mvpa2/generators/permutation.py

```diff
@@ -94,6 +97,10 @@ def _get_pcfg(self, ds):
         return get_limit_filter(self._limit, collection)
 
 
+    def get_chunk_attr(self, ds):
```

Comment:
like ```_assign_chunk_attr```. I would have different approach though to make it more explicit... easier to do than to say -- I will send you a PR shortly -- you decide either to accept ;)